sample/event-read-fifo.c

```diff
@@ -158,5 +165,16 @@ main(int argc, char **argv)
 #endif
 	libevent_global_shutdown();
 	return (0);
+
+err:
+	event_base_free(base);
+#ifdef _WIN32
+	CloseHandle(socket);
+#else
+	close(socket);
+	unlink(fifo);
+#endif
+	libevent_global_shutdown();
```

Comment:
```
/home/runner/work/libevent/libevent/dist/libevent-2.2.1-alpha-dev/sample/event-read-fifo.c:177:2: error: type defaults to ‘int’ in declaration of ‘libevent_global_shutdown’ [-Werror=implicit-int]
```

Reply:
oops, here is the fix: 02f03236f6a8a1fc6e34becc2a7767559e72613a